xml/mdns_device-info_txt.xml

```diff
@@ -12,110 +12,462 @@
      The number specified after osxvers= is equivalent to the major
      version of OS X plus 4. E.g. osxvers=13 means OS X 10.9
    -->
+  <fingerprint pattern="^osxvers=17">
+    <description>OS X 10.13 (High Sierra)</description>
+    <example>osxvers=17</example>
+    <param pos="0" name="os.vendor" value="Apple"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:apple:mac_os_x:10.13"/>
+    <param pos="0" name="os.family" value="Mac OS X"/>
+    <param pos="0" name="os.product" value="Mac OS X"/>
+    <param pos="0" name="os.version" value="10.13"/>
+  </fingerprint>
+  <fingerprint pattern="^osxvers=16">
+    <description>OS X 10.12 (Sierra)</description>
+    <example>osxvers=16</example>
+    <param pos="0" name="os.vendor" value="Apple"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:apple:mac_os_x:10.12"/>
+    <param pos="0" name="os.family" value="Mac OS X"/>
+    <param pos="0" name="os.product" value="Mac OS X"/>
+    <param pos="0" name="os.version" value="10.12"/>
+  </fingerprint>
+  <fingerprint pattern="^osxvers=15">
+    <description>OS X 10.11 (El Capitan)</description>
+    <example>osxvers=15</example>
+    <param pos="0" name="os.vendor" value="Apple"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:apple:mac_os_x:10.11"/>
+    <param pos="0" name="os.family" value="Mac OS X"/>
+    <param pos="0" name="os.product" value="Mac OS X"/>
+    <param pos="0" name="os.version" value="10.11"/>
+  </fingerprint>
   <fingerprint pattern="^osxvers=14">
     <description>OS X 10.10 (Yosemite)</description>
     <example>osxvers=14</example>
     <param pos="0" name="os.vendor" value="Apple"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:apple:mac_os_x:10.10"/>
```

Comment:
I wonder why this wasn't added during the first run?  If you did this manually, it will be removed as part of future runs.

Reply:
It was already there, just in a difference position.